Common/src/main/java/com/techarium/techarium/block/entity/selfdeploying/SimpleFluidContainer.java

```diff
@@ -0,0 +1,168 @@
+package com.techarium.techarium.block.entity.selfdeploying;
+
+import com.google.common.base.Suppliers;
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import com.techarium.techarium.platform.CommonServices;
+import net.minecraft.core.Holder;
+import net.minecraft.core.Registry;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.level.material.Fluid;
+import net.minecraft.world.level.material.Fluids;
+
+import java.util.Optional;
+
+/**
+ * A module that store a fluid.
+ */
+public class SimpleFluidContainer {
+	public static final SimpleFluidContainer EMPTY = new SimpleFluidContainer(0);
+
+	// This can be changed if we want to use a different internal representation format
+	public static final long BUCKET_CAPACITY = 81000;
```

Comment:
I am concerned about this. What is the reason behind changing it to not be loader dependant ?
The problem I see with that is in forge, the bucket capacity is 1000, so when you'll retrieve, from a fluid tank that is not from techarium, you'll do something like tank.retrieve(BUCKET_CAPACITY) and you'll remove 81 buckets instead of only 1. and the same will happen on the otherside, when they want to retrieve 1 bucket, they'll retrieve 1000 instead of 81000

Reply:
We should use one format internally in my opinion, rather than relying on what the mod loaders use. When interacting with loader specific APIs, we should have functions that can internally convert the format to mB when needed.

Reply:
That way we only use one number, instead of always getting the bucket size from the platform, but it's still consistent.